### PR TITLE
update patch calendar

### PR DESCRIPTION
**Pull request by cpanato**
- update patch calendar

/hold for discussion/clarifications

/assign @saschagrunert @justaugustus @puerco @palnabarun @Verolop
/cc @kubernetes/release-engineering

### CODE REVIEW

File: content/en/releases/patch-releases.md [hunk 1]
```diff
@@ -78,13 +78,23 @@ releases may also occur in between these.
 
 | Monthly Patch Release | Cherry Pick Deadline | Target date |
 | --------------------- | -------------------- | ----------- |
-| April 2022            | 2022-04-08           | 2022-04-13  |
-| May 2022              | 2022-05-13           | 2022-05-18  |
+| May 2022              | 2022-05-20           | 2022-05-24  |
```
Comment (cpanato) [inline content/en/releases/patch-releases.md:81]: move one week ahead due to kubecon

so the CP deadline will be on Friday (the last day of Kubecon) and the cut in the following week

File: content/en/releases/patch-releases.md [hunk 2]
```diff
@@ -78,13 +78,23 @@ releases may also occur in between these.
...
 | June 2022             | 2022-06-10           | 2022-06-15  |
 | July 2022             | 2022-07-08           | 2022-07-13  |
+| August 2022           | 2022-08-12           | 2022-08-16  |
 
 ## Detailed Release History for Active Branches
 
+### 1.24
+
+Next patch release is **1.24.1**
```
Comment (cpanato) [inline content/en/releases/patch-releases.md:90]: added the 1.24.1 for this month

File: content/en/releases/patch-releases.md [hunk 3]
```diff
@@ -78,13 +78,23 @@ releases may also occur in between these.
...
+
+End of Life for **1.24** is **2023-09-29**
```
Comment (cpanato) [inline content/en/releases/patch-releases.md:92]: Need to review the correct EOL for 1.24

**Review by saschagrunert — APPROVED** [review]
/lgtm

**Comment by sftim** [comment]
What are the prerequisites for un-holding this?

**Comment by cpanato** [comment]
none :)
sascha approved so we are good 
/hold cancel

**Pull request merged**